CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ Version History
     * All Version bumps are required to update this file as well!!
 ----
 
+* 18.6.0 Removed job search version(s) prior to version 3.
```

Comment:
You should also put in here what would break for consumers of this. For example, this also renames the V3 job search to just job search, or something.